cmd/livepeer/starter/starter.go

```diff
@@ -1494,6 +1524,73 @@ func parseEthKeystorePath(ethKeystorePath string) (keystorePath, error) {
 	return keystore, nil
 }
 
+func parsePricePerUnit(pricePerUnitStr string) (*big.Rat, string, error) {
+	match := pricePerUnitRex.FindStringSubmatch(pricePerUnitStr)
```

Comment:
nit: I'd probably inline this regexp and add a comment. Otherwise, it's very hard to reason about what's going on.
```suggestion
        pricePerUnitRex := regexp.MustCompile(`^(\d+(\.\d+)?)([A-z][A-z0-9]*)?$`)
	match := pricePerUnitRex.FindStringSubmatch(pricePerUnitStr)
```